src/components/foodbox/Foodbox.jsx

```diff
@@ -0,0 +1,39 @@
+
+import './Foodbox.css';
+
+
+export default function FoodBox({ name, calories, image, quantity }) {
+        return (
+            <div className="Foodbox">
+            <div className="box">
+                <article className="media">
+                    <div className="media-left">
+                    <figure className="image is-64x64">
+                        <img src={image} alt='img' />
+                    </figure>
+                    </div>
+                    <div className="media-content">
+                    <div className="content">
+                        <p>
+                        <strong>{name}</strong> <br />
+                        <small>{calories} cal</small>
+                        </p>
+                    </div>
+                    </div>
+                    <div className="media-right">
+                    <div className="field has-addons">
+                        <div className="control">
+                        <input className="input" type="number" value="1" />
```

Comment:
En el input tenéis puesto el **value** fijo a 1, por tanto nunca va a cambiar. Este input necesita un **onChange** que modifique una variable (llamada **quantity** por ejemplo) en el estado del componente, por lo que **FoodBox** debería ser tipo _class_. Entonces el input va establecer su valor a partir de esta variable, que empieza siendo 0.